<?xml version="1.0"?>
<launch>
  <node pkg="ros_dcdc_usb" type="dcdc_usb_node" name="dcdc_usb_node" output="screen">      
     <!-- Critical low battery level (V)  -->
     <param name="criticalLowInput" value="11.5"/>
     
     <!-- Warning low battery level (V)  -->
     <param name="warningLowInput" value="12.2"/>

     <!-- Output level (V) [Default 10%] -->
     <param name="output" value="19.0"/>
 
     <!-- Output tollerance (V) [Default 0.3] -->
     <param name="outputToll" value="0.3"/>
   </node>   
</launch>
